views/partials/bodytemplate.pug

```diff
@@ -0,0 +1,81 @@
+body
+  h2 Welcome to the League of Legends API
+  div(class="mainBoi")
+    div(class="leftDocs")
+      div(class="leftTabs selected" id="getAll" onclick="selected('getAll')")
+        span GET
+        span /lol
```

Comment:
This should be returning data, not the documentation

![image](https://user-images.githubusercontent.com/5775716/84073794-78cfcd00-a99f-11ea-8699-d66bb6820557.png)